package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "futurice-cv",
+  "version": "1.0.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "jquery": {
+      "version": "2.2.4",
+      "resolved": "https://registry.npmjs.org/jquery/-/jquery-2.2.4.tgz",
```

Comment:
The version in package-lock.json (2.2.4) does not match the version in package.json (3.3.1). Is it bad?